CHANGELOG.md

```diff
@@ -5,27 +5,39 @@ CHANGELOG
 -----
 
 **ENHANCEMENTS**
-- Add support for updating `SharedStorage` configuration.
+- Add support for AWS Trainium instances.
+- Add support for Slurm Accounting.
+- Add support for adding and removing shared storages at cluster update by updating `SharedStorage` configuration.
 - Add new configuration parameter `DeletionPolicy` for EFS and FSx for Lustre shared storage
   to support storage retention on deletion.
-- Enable server-side encryption for the EcrImageBuilder SNS topic created when deploying ParallelCluster API and used to notify on docker image build events.
+- Add possibility to specify multiple instance types for the same compute resource.
```

Comment:
This is one of the top features, I'd keep it as first, then adding/removing shared storage, then others.

Reply:
I agree, thanks. Will keep the DeletionPolicy change close to dynamic file systems mounting because they are strictly related